django_app/redbox_app/redbox_core/admin.py

```diff
@@ -82,26 +82,35 @@ class CitationInline(admin.StackedInline):
     extra = 1
 
 
-class ChatMessageAdmin(admin.ModelAdmin):
-    list_display = ["text", "role", "get_user", "chat", "route", "created_at"]
+class ChatMessageAdmin(ExportMixin, admin.ModelAdmin):
+    list_display = ["short_text", "role", "get_user", "chat", "route", "created_at"]
     list_filter = ["role", "route", "chat__user"]
     date_hierarchy = "created_at"
     inlines = [CitationInline]
+    readonly_fields = ["selected_files", "source_files"]
 
     @admin.display(ordering="chat__user", description="User")
     def get_user(self, obj):
         return obj.chat.user
 
+    @admin.display(description="text")
+    def short_text(self, obj):
```

Comment:
this is just for display purposes, we dont need to see the full text in the summary